### PR TITLE
Hide icon

### PR DESCRIPTION
**Pull request by ianwitherow**
Lets you hide the calendar icon if all you want is the text box. This would have to be used with openOnInputFocus=true.

### CODE REVIEW

**Comment by ianwitherow** [comment]
Implements #11

File: dist/Calendar.js [hunk 1]
```diff
@@ -274,8 +275,13 @@ module.exports = React.createClass({displayName: "exports",
         if(moment.locale() === 'de')
           todayText = 'Heute';
 
+		  var calendarClass = cs({
+				 'input-calendar-wrapper': true,
+				 'icon-hidden': this.props.hideIcon
+			 });
+
```
Comment (Rudeg) [inline dist/Calendar.js]: indents

File: src/Calendar.js [hunk 2]
```diff
@@ -315,9 +327,7 @@ module.exports = React.createClass({
                     placeholder={this.props.placeholder}
                     readOnly={readOnly} />
 
-                <span onClick={this.toggleClick} className="icon-wrapper calendar-icon">
-                    <i className={iconClass}></i>
-                </span>
+						  {calendarIcon}
```
Comment (Rudeg) [inline src/Calendar.js]: indents

File: styles/input-calendar.css [hunk 3]
```diff
@@ -21,6 +21,9 @@
     user-select: none;
     z-index: 2;
 }
+.input-calendar-wrapper.icon-hidden {
+	left: 0;
```
Comment (Rudeg) [inline styles/input-calendar.css]: indents

**Comment by Rudeg** [comment]
Great, thanks!!!

**Pull request merged**